packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1814,133 +1853,6 @@ function mountIncompleteClassComponent(
   );
 }
 
-function mountIndeterminateComponent(
-  _current: null | Fiber,
-  workInProgress: Fiber,
-  Component: $FlowFixMe,
-  renderLanes: Lanes,
-) {
-  resetSuspendedCurrentOnMountInLegacyMode(_current, workInProgress);
```

Comment:
I think I missed this. Should probably call this in updateFunctionComponent

Reply:
hmm that doesn't really make sense though b/c why would the function component be "remounting". Can't add this to updateFunctionComponent because we normally don't want to unlink the fibers so it's just when there is a semantic mount on already linked fibers which doesn't seem to make sense for function components that aren't Indeterminant